clubs/berkeley_high_plan.md

```diff
@@ -0,0 +1,157 @@
+# Berkeley High S1 Club Plan
+
+_**The following is a club plan for the first semester of the Berkeley High Hack Club, coordinated with the school's calendar. We recommend that you make a copy for yourself that conforms to your school schedule.**_
+
+<table>
+    <thead>
+        <tr>
+            <th>Week</th>
+            <th>Meeting</th>
+            <th>Goal</th>
+            <th>Workshop</th>
+            <th>Schedule</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>1</td>
+            <td>Intro</td>
+            <td>Teach students what they will do in Hack Club and get acquainted with members.</td>
+            <td>None</td>
+            <td>1. Wait for members to trickle in <br> 2. Name game <br> 3. Club leader presentation (what is hack club?) <br> 4. Chat with members</td>
+        </tr>
+        <tr>
+            <td>2</td>
+            <td>Workshop</td>
+            <td>Get members through the first workshop and teach them how easy it is to code!</td>
+            <td><a href="https://workshops.hackclub.com/personal_website/README.md">Personal Website</td>
+            <td>1. Recap Last meeting <br> 2. Guide members through first part of workshop <br> 3. Let members go through workshop, offer help</td>
```

Comment:
FYI, you should be able to break this into multiple lines (instead of using `<br>`) now that you're using HTML tables.